satnogsclient/observer/orbital.py

```diff
@@ -1,56 +1,106 @@
 # -*- coding: utf-8 -*-
-import sys
-
 from datetime import datetime
-
-import ephem
 import pytz
+import ephem
```

Comment:
Many major python projects require alphabetically ordering. Here [1] is the most common style guide for module importing (not mozilla specific, many projects follow this). Also there is no need to change the order here (doesn't make any difference to the commit).

[1] https://mozweb.readthedocs.org/en/latest/reference/python-style.html#import-statements